### ISSUE TITLE
Maximize nodes reflects the drawing and not the canvas

### ISSUE THREAD

**Issue by F-Burning** [opentoonz/opentoonz#4652]
### Is your feature request related to a problem? Please describe

When you maximize a node to see its thumbnail, it reflects the entire canvas, so if the drawing is tiny, you won't be able to see it reflected.

### Describe the solution you'd like

I would like the thumbnail to reflect the drawing, so that I can see it on it, and not the canvas

**Comment by RodneyBaker** [comment]
Such a change would need to be optional as the full canvas view allows the user to see the thumbnail relative to each other which give hints as to how they all work together.  In cases where the thumbnail would change size relative to itself that would be lost.

This is outside the scope of this request but due to other areas where previewing can be troublesome it would be good to have a 'hover' capability where when the user hovers for an extended period of time over the preview/thumbnail a larger preview appears at that location.  If implemented that hover preview might have a Shift + hover capability that would allow zooming in to  a shapes bounding box.  Note that I do not know if Qt has the generalized code for doing such hover-previews which would probably be a prerequisite to ensure that remains future proof.

**Comment by F-Burning** [comment]
Yes, making it optional is a better idea.

For example a very small drawing looks like this:

![02](https://user-images.githubusercontent.com/74098734/207921511-862802ba-2a74-4fc4-b214-de3557271373.jpg)

And the target would be something like this:

![02](https://user-images.githubusercontent.com/74098734/207921518-55edcd79-14d0-47c5-bac1-ec7a366bf02f.png)

I don't know if I understood you correctly because of the problem of the size of the thumbnails, but my idea is that the thumbnails maintains its size, but the content is in focus so that it can be easily distinguished as in the example above.

**Comment by artisteacher** [comment]
Vector levels already do this - since they have an infinite canvas. I'm all for it - it would make things more consistent. And should be extended to layer thumbnails & etc.

> **I'd have no issue with the default being to zoom the the hover preview to the bounding box and the assisted key hover be to present the whole canvas.

Love this!

**Comment by DarrenTAnims** [comment]
Agreed. I think that the relative location/size to the full drawing is less useful in the schematic view, so I think that if only the bounding box location for the actual strokes was shown, it would be much more useful.
